README.md

```
@@ -27,6 +27,20 @@ Beta (Beta Docker Image): https://newsbeta.20090526.xyz
 
 https://github.com/user-attachments/assets/29414c5d-3b2f-420d-93c0-95c14a15bbb7
 
+## Notes:
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: ':'

(MD026, no-trailing-punctuation)